packages/language-server/src/test/rename.test.ts

```diff
@@ -0,0 +1,297 @@
+import { TextDocument } from 'vscode-languageserver-textdocument'
+import { handleRenameRequest } from '../MessageHandler'
+import { WorkspaceEdit, RenameParams, Position } from 'vscode-languageserver'
+import * as assert from 'assert'
+import { getTextDocument } from './helper'
+
+function assertRename(
+  expected: WorkspaceEdit,
+  document: TextDocument,
+  newName: string,
+  position: Position,
+): void {
+  const params: RenameParams = {
+    textDocument: document,
+    newName: newName,
+    position: position,
+  }
+
+  const renameResult: WorkspaceEdit | undefined = handleRenameRequest(
+    params,
+    document,
+  )
+
+  assert.ok(renameResult !== undefined)
```

Comment:
You can also use `assert.equal` here (gives better error messages)
```suggestion
  assert.equal(renameResult, undefined)
```